primitives/src/sentry.rs

```diff
@@ -214,6 +214,12 @@ pub struct SpenderResponse {
     pub spender: Spender,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct AllSpendersResponse {
+    pub spenders: HashMap<Address, Spender>,
```

Comment:
You can add the Pagination struct for now and later we will make use of it.
For now it will be 1 page only.
```suggestion
    pub spenders: HashMap<Address, Spender>,
    #[serde(transparent)]
    pub pagination: Pagination,
```